CHANGELOG.md

```diff
@@ -311,10 +323,11 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 > 16 September 2025
 
+- fix: home number fields are added to the draft despite not being modified  [`#8704`](https://github.com/opengovsg/FormSG/pull/8704)
 - feat: save draft v1.0 [`#8091`](https://github.com/opengovsg/FormSG/pull/8091)
 - build: merge release v6.240.0 to develop [`#8695`](https://github.com/opengovsg/FormSG/pull/8695)
 - build: release v6.240.0 [`#8693`](https://github.com/opengovsg/FormSG/pull/8693)
-- chore: bump version to v6.241.0 [`8572128`](https://github.com/opengovsg/FormSG/commit/8572128021278843f40d95dd30722a2f81370e5d)
+- chore: bump version to v6.241.0 [`13345f3`](https://github.com/opengovsg/FormSG/commit/13345f3eabe3c954d90b8ec10ca4e15c6b709b68)
```

Comment:
one downside of a non-linear git history: commit hashes are different on different branches